docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,971 @@
+
+The following command will show the deployed pods of the newly created region.
+
+Depending on your chosen `clusterSize` you should see that the **failback** deployment contains some Zeebe instances being ready and others unready. Those unready instances are sleeping indefinitely and is the expected behaviour.
+This behaviour stems from the **failback** mode since we still have the temporary **failover**, which acts as replacement for the lost region.
+
+For example in the case of `clusterSize: 8`, you find 2 active Zeebe brokers and 2 unready brokers in the newly created region.
+
+```bash
+kubectl --context $CLUSTER_1 get pods -n $CAMUNDA_NAMESPACE_1
+```
+
+Port-forwarding the Zeebe Gateway via `kubectl` and printing the topology should reveal that the **failback** brokers have joined the cluster.
+
+```bash
+ZEEBE_GATEWAY_SERVICE=$(kubectl --context $CLUSTER_0 get service --selector=app\.kubernetes\.io/component=zeebe-gateway -o jsonpath='{.items[0].metadata.name}' -n $CAMUNDA_NAMESPACE_0)
+kubectl --context $CLUSTER_0 port-forward services/$ZEEBE_GATEWAY_SERVICE 26500:26500 -n $CAMUNDA_NAMESPACE_0
+zbctl status --insecure --address localhost:26500
+```
+
+</div>
+  </TabItem>
+  <TabItem value="step2" label="Step 2">
+
+#### Pause Elasticsearch Exporters and Operate / Tasklist
+
+<StateContainer
+current={<Nine viewBox="140 40 680 500" />}
+desired={<Ten viewBox="140 40 680 500" />}
+/>
+
+<div>
+
+#### Current
+
+You currently have the following setups:
+
+- Healthy Camunda Platform
+- Camunda Platform in failover mode within the same region as the healthy setup
+- Camunda Platform in failback mode within a newly created region
+
+#### Desired
+
+:::warning
+
+This step is very important to minimise the risk of loosing any data when restoring the backup in the new region.
+
+There remains a small chance of losing some data in Elasticsearch (in turn in Operate and Tasklist). This is because Zeebe might have exported some records to the failover Elasticsearch in `REGION_0`, but not to the main Elasticsearch in `REGION_0` before pausing the exporters. So those records are not included in the `REGION_0` Elasticsearch backup when the new `REGION_1` Elasticsearch is restored from the `REGION_0` backup, the new region is missing those records and Zeebe does not re-export them.
+
+:::
+
+You are preparing everything for the newly created region to take over again to restore the benefits of a dual-region setup.
+
+For this, you need to stop the Zeebe exporters to not export any new data to Elasticsearch, so you can create a backup.
+
+Additionally, you need to scale down Operate and Tasklist. This will result in users not being able to interact with the Camunda Platform anymore and is required to guarantee no new data is imported to Elasticsearch.
+
+:::note
+
+That this **does not** affect processing of process instances in any way. The impact is that some information about the affected instances might not be visible in Operate.
```

Comment:
process information in Operate / Tasklist.
Like you could have a small discrepancy between what you see and what actually has happened in a process since you may be missing some information as it was skipped.

Reply:
changed to

```
This **does not** affect the processing of process instances in any way. The impact is that process information about the affected instances might not be visible in Operate and Tasklist.
```